paystell-frontend/src/app/PaymentLinkScreen/page.tsx

```diff
@@ -0,0 +1,131 @@
+        <div className="text-center flex-grow">
+          <h2 className="text-sm font-semibold text-[#2C384F]">Make payments with PayStell</h2>
+          <p className="text-xs text-[#2C384F]">Your fastest Decentralized payment gateway</p>
+        </div>
+      </header>
+
+      {/* Main container */}
+      <div className="w-full max-w-[750px] mx-auto bg-white rounded-lg shadow-lg p-6 sm:p-8 space-y-6">
+
+        {/* Product Information Container */}
+        <div className="flex flex-col space-y-2 bg-white border-b border-gray-300 p-4">
+          <div className="flex justify-between text-sm items-center">
+            <div className="flex items-center space-x-2">
+              <Image
+                src={Product}  // Relative path
+                alt="Product Image"
+                width={80} 
+                height={80} 
+              />
+              <div className="flex flex-col">
+                <span className="text-[14px]">Product Name:</span>
+                <span className="font-semibold text-[14px]">SKU/ID:</span>
+              </div>
+            </div>
+            <div className="flex flex-col items-end">
+              <span className="text-[14px]">Trouser</span>
+              <span className="font-semibold text-[16px]">17639041</span>
+            </div>
+          </div>
+        </div>
+
+      {/* Subtotal Container and Gas Fee */}
+      <div className="flex flex-col space-y-2 bg-white border-b border-gray-300 p-4">
+
+          {/* Sub Total */}
+          <div className="flex justify-between text-sm">
+            <span className="font-semibold text-[16px]">Sub Total:</span>
+            <span className="font-semibold text-[16px]">0.8 XML</span>
+          </div>
+
+          {/* Gas Fee */}
+          <div className="flex items-center justify-between text-xs">
+            <div className="flex items-center">
+              <span className="">Gas Fee</span>
+              <button className="ml-1 w-4 h-4 flex items-center justify-center rounded-full text-black text-xs border-2 border-black">
+                !
+              </button>
+            </div>
+            <span className="font-semibold text-[#FF3131] text-[12px]">0.2 XML</span>
+          </div>
+        </div>
+
+        {/* Total Amount Container */}
+        <div className="flex justify-between text-lg font-semibold bg-white border-b border-gray-300 p-4">
+          <span className="text-[16px]">Total Amount:</span>
+          <span className="text-[36px]">1 XML</span>
+        </div>
+
+        {/* Container for the payment button and the instruction */}
+        <div className="flex flex-col items-center w-full mt-10 space-y-2">
+          {/* Payment button */}
+          <Dialog.Root>
+            <Dialog.Trigger asChild>
+              <button className="w-[300px] h-[60px] sm:w-[300px] bg-[#009EFF] text-white px-8 py-3 rounded-md hover:bg-[#007ACC] transition-colors duration-300">
+                Pay
+              </button>
```

Comment:
Use button component
```suggestion
                <Button size="xl" >
                    Pay
                </Button>
```